### PR TITLE
Update the SageMaker Model Monitoring module with model bias monitoring.

### PR DESCRIPTION
**Pull request by clokep**
## Describe your changes

Expands the module from #117 & #125 (`sagemaker-model-monitoring-module`) to include monitoring model bias.

Similar to #117 & #125 this is somewhat difficult to test as you need a SageMaker Pipeline with the proper check steps & also ground truth for the data.

## Issue ticket number and link

N/A

## Checklist before requesting a review

- [x] I updated `CHANGELOG.MD` with a description of my changes
- [x] If the change was to a module, I ran the code validation script (`scripts/validate.sh`)
- [x] If the change was to a module, I have added thorough tests
- [x] If the change was to a module, I have added/updated the module's README.md
- [x] If a module was added, I added a reference to the module to the repository's README.md
- [x] I verified that my code deploys successfully using `seedfarmer apply`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: AIOpsModuleTestsInfrastruct-doMLXEYsnmxr
* Commit ID: 663754038cce2e6ccee9969e9decec4c1a9b1bea
* Result: SUCCEEDED
* [Build Logs](https://ythfngxmh6.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=88be41bc-9112-48a6-be4a-37f71c5e2964%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by malachi-constant** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: AIOpsModuleTestsInfrastruct-doMLXEYsnmxr
* Commit ID: 43b2ab3cd6165c11648d0ac7ae4bba9534dc4bef
* Result: SUCCEEDED
* [Build Logs](https://ythfngxmh6.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=244c2ea9-47cc-477c-8f38-05cef4b07844%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: modules/sagemaker/sagemaker-model-monitoring/README.md [hunk 1]
```diff
@@ -68,15 +74,24 @@ N/A
...
 - `model-quality-probability-threshold-attribute`: The threshold for the class probability to be evaluated as a positive result.
 
+##### Model Bias Monitoring Job Parameters
+
+###### Optional
+
+- `model-bias-checkstep-analysis-config-prefix`: The S3 prefix in `model-artifacts-bucket-arn` which contains the output from the [Clarify Check Step](https://docs.aws.amazon.com/sagemaker/latest/dg/build-and-manage-steps.html#step-type-clarify-check) used for model bias.
+- `model-bias-features-attribute`: The attributes of the input data that are the input features.
+- `model-bias-inference-attribute`: The attribute of the input data that represents the ground truth label.
+- `model-bias-probability-attribute`: In a classification problem, the attribute that represents the class probability.
```
Comment (clokep) [inline modules/sagemaker/sagemaker-model-monitoring/README.md:92]: I need to check if it makes sense for these to be separate parameters from the model quality job.

Comment (clokep) [reply modules/sagemaker/sagemaker-model-monitoring/README.md:92]: I think we'll just go with this for now to leave it flexible.

**Review by kukushking — APPROVED** [review]
Thank you!

**Pull request merged**